src/internal/components/chart-series-marker/styles.scss

```diff
@@ -6,12 +6,15 @@
 @use '../../styles' as styles;
 @use '../../styles/tokens' as awsui;
 
+$marker-width: 1.4 * styles.$base-size;
+$marker-margin-right: awsui.$space-xxs;
```

Comment:
Why the variable for the right margin is needed? (And why there is no variable for the top margin?)

Reply:
I'm using them in ChartSeriesDetails, [here](https://github.com/cloudscape-design/components/pull/1644/files/a9ad08dfb8e982f84e72ba623f1ac1b049857d78#diff-759e11b0626a727a2deb0cfbe10103be7c565cffc8194fb3cb06a189920601b7R48), to make sure non-expandable sub-items are left-aligned with their parents.